README.md

```diff
@@ -56,29 +52,6 @@ plugins: [
   },
 ]
 ```
-
-### Public scraping for a user's profile
```

Comment:
can we keep those deprecated for reference please? i dont have any other docs page :D